assets/src/components/nav.tsx

```diff
@@ -26,46 +23,30 @@ const Nav: React.FC<Props> = ({ children }) => {
           <div className="l-nav__app-content">{children}</div>
         </div>
       )
-    case "mobile_landscape_tablet_portrait":
-      return (
-        <div className="l-nav--medium">
-          <div
-            className="l-nav__nav-bar l-nav__nav-bar--left"
-            hidden={isViewOpen}
-          >
-            <LeftNav
-              toggleMobileMenu={() => dispatch(toggleMobileMenu())}
-              defaultToCollapsed={true}
-              dispatcherFlag={readDispatcherFlag()}
-              closePickerOnViewOpen={true}
-            />
-          </div>
-          <div className="l-nav__app-content">{children}</div>
-        </div>
-      )
-    case "tablet":
-      return (
-        <div className="l-nav--medium">
-          <div className="l-nav__nav-bar l-nav__nav-bar--left">
-            <LeftNav
-              toggleMobileMenu={() => dispatch(toggleMobileMenu())}
-              defaultToCollapsed={true}
-              dispatcherFlag={readDispatcherFlag()}
-            />
-          </div>
-          <div className="l-nav__app-content">{children}</div>
-        </div>
-      )
     default:
       return (
         <div className="l-nav--wide">
           <>
-            <div className="l-nav__nav-bar l-nav__nav-bar--top">
+            <div
+              className="l-nav__nav-bar l-nav__nav-bar--top"
+              hidden={
+                isViewOpen && deviceType === "mobile_landscape_tablet_portrait"
+              }
```

Comment:
question(non-blocking): I pulled this down locally and it worked fine, but it didn't work for me in dev green, any ideas why?

Reply:
Yeah I just ran into the same thing with design review, I'm pulling my hair out trying to figure it out.

Reply:
Ah, the deploy action was marked as successful but the circuit breaker actually rolled it back. Gonna retry.

Reply:
Wuttt... why would it do that?

Reply:
I think it hit the ECS deploy circuit breaker that eventually gives up on failing deploys (it was having issues connecting to the DB). That's great and all but it isn't well-integrated into our deploy action so it basically sees that the deploy finished without detecting the particular error state it hit.